<launch>

	<node pkg="furniture" name="table_tracker" type="table_tracker" respawn="true" output="screen">

		<remap from="/points" to="/camera/rgb/points" />
    		<param name="sensor_frame" type="string" value="/openni_rgb_optical_frame" />

		<!--param name="multiplier" type="double" value="0.0075" /-->
		<param name="multiplier" type="double" value="0.0075"/>
		<!--param name="downsample" type="double" value="0.01" /-->
		<param name="downsample" type="double" value="0.01" />
	</node> 

</launch>
